IoProtocol.h

```diff
@@ -504,22 +504,19 @@ typedef struct {
 
 //-----------------------------------------------------------------------------
 // Request for Multi IO modules:
-typedef struct {
-	//bitfield: mode
-	INT8U i8uCalibrationMode;
-	//channels to calibrate
-	INT8U i8uChannels;
-	//specifies point in lookupTable
-	INT8U i8uPoint;
-	INT16S i16sCalibrationValue;
-} SMioCalibrationRequestData;
+struct calibration_data {		// IOP_TYP1_CMD_DATA6
+	u8 mode;
+	u8 channel;
+	u8 refPoint;
+	s16 val;
+} __attribute__((__packed__));
```

Comment:
We're losing information here which is encoded in the variable names and in the code comments that are being deleted. That's not good, the information should be retained in code comments at least.

Reply:
Ok